tests/test_sim_client.py

```diff
@@ -203,7 +203,7 @@ def determine_expected_state(self, topic):
                         expected_state[variable] = False
                 elif var["idl_type"] == "float":
                     lower_limit, upper_limit = var["limits"]
-                    if var["unit"] in ["deg_C", "unitless", "bar", "%"]:
+                    if var["unit"] in ["deg_C", "unitless", "bar", "%", "h"]:
```

Comment:
I gently suggest a frozenset that you construct outside of the loop.
```
units_with_limits = frozenset(("deg_C", "unitless", "bar", "%", "h"))
...
if var["unit"] in units_with_limits:
    ...
```
Though what you have is also fine. Less efficient, but it's just a unit test.

Reply:
Done